torchvision/prototype/transforms/_augment.py

```diff
@@ -110,15 +114,32 @@ def forward(self, *inpts: Any) -> Any:
         return super().forward(sample)
 
 
+def _mixup_onehotlabel(inpt: features.OneHotLabel, lam: float) -> features.OneHotLabel:
+    if inpt.ndim < 2:
+        raise ValueError("Need a batch of one hot labels")
+    output = inpt.clone()
+    output = output.roll(1, -2).mul_(1 - lam).add_(output.mul_(lam))
+    return features.OneHotLabel.new_like(inpt, output)
+
+
 class RandomMixup(_BaseMixupCutmix):
     def _get_params(self, sample: Any) -> Dict[str, Any]:
         return dict(lam=float(self._dist.sample(())))
 
     def _transform(self, inpt: Any, params: Dict[str, Any]) -> Any:
-        if isinstance(inpt, features._Feature):
-            return inpt.mixup(**params)
-        else:
-            return inpt
+        lam = params["lam"]
```

Comment:
@datumbox I removed mixup/cutmix from functional part and put it into Transforms. Is it that what you had in mind ?

Reply:
Yes, looks good.